### DISCUSSION TITLE
How can I find my verify_fp?

### DISCUSSION THREAD

**Discussion by diracolam** [davidteather/TikTok-Api#1009] [Q&A] [answered]
i am looking in my tiktok cookies and cannot find a cookie labeled verify_fp.

**Comment by Monredo** [comment, marked as answer]
Go to TikTok and open Dev Tool > Application > Storage > Cookies. Then copy s_v_web_id value and paste it to verifyFp in the following sample code. You can see what you need to do to get s_v_web_id in the YouTube video.
